composables/httpd.ts

```diff
@@ -0,0 +1,22 @@
+const PER_PAGE = 100
```

Comment:
Ultimately this whole file is obsolete and should be removed by moving (some of) this logic inside `plugins/httpd-client.ts`, but I added some more review comments here to share some insight.

Reply:
👍, deleted the file.